### ISSUE TITLE
Genentech ID mismatch

### ISSUE THREAD

**Issue by jlkravitz** [hammerlab/multi-omic-urothelial-anti-pdl1#1]
For those not using the provided python code to load the data, note that the file `tcga_subtypes_id_map.csv` maps a genentech ID to a patient ID of 2397, even though that doesn't exist in `clinical.csv`. I believe this is a mismatch and that that patient ID should be 2937, which does exist in `clinical.csv`.

**Comment by tavinathanson** [comment]
Yup, that's right. For those using the provided Python code, that is addressed in https://github.com/hammerlab/multi-omic-urothelial-anti-pdl1/blob/master/analyses/utils/data.py#L453
